app/controllers/schools/base_controller.rb

```diff
@@ -9,6 +9,8 @@ class Schools::BaseController < ApplicationController
   after_action :verify_authorized
   after_action :verify_policy_scoped
 
+  helper_method :breadcrumbs
```

Comment:
Should we unit test this? If so, should we extract it to a helper to make our lives easier in tests?

Reply:
Not quite sure how we would extract this, as it calls `active_school` and `active_cohort`

Reply:
Mhmm, and the difficulty of extraction means we don't want to test it too? :P 

If I were to extract it I would make a helper with breadcrumbs method that takes a school parameter and a cohort parameter, and builds breadcrumb content from that. And then in the controller all you do is pass the active school and/or cohort in. Or maybe in the view you pass the relevant school / cohort into the helper method. And then you can test the logic in helper easily.

If it sounds like too much work, I guess we can leave it untested, or maybe 'tested implicitly via percy'.

Reply:
No, it just means that I don't know how to extract it. Thanks for the suggestion, I'll take a look